blocks/identity-block/package.json

```diff
@@ -30,8 +30,12 @@
   },
   "dependencies": {
     "@arc-fusion/prop-types": "^0.1.5",
-    "@arc-publishing/sdk-identity": "^1.79.0",
+    "@arc-publishing/sdk-identity": "^1.88.0",
```

Comment:
@LauraPinilla Is there any reason why this dependency been added in both block's package.json and outer package.json? In subscriptions-blocks it's still v1.55.0.